__tests__/nc_news_db.test.js

```diff
@@ -80,3 +89,37 @@ describe("GET /api/articles/:articles_id", () => {
 			});
 	});
 });
+
+describe("GET /api/articles", () => {
+	it("status: 200, should respond with all articles and their properties", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body }) => {
+				const bodyArray = body.articles;
+				bodyArray.forEach((article) => {
```

Comment:
I would check the length of the array before your forEach - otherwise the test could pass if the array is empty